internal/commands/user/inputs.go

```diff
@@ -0,0 +1,110 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+func validAuthProviderTypes() []interface{} {
```

Comment:
[optional/future drive-by] might be worth having this be a pre-computed `var` eventually:

```go
var validAuthProviderTypes = func() []interface{} { /* make the []interface{} here */}()
```

since it's not changing much